### PR TITLE
Update: Add participants

### PR DESCRIPTION
**Pull request by vimode**
- [x] I have read and understand the [Contributing.md](../blob/master/CONTRIBUTING.md) guidelines
- [x] I do not intend to make low-value changes, just for swag. I am aware that my PR could otherwise be flagged as `spam` or `invalid` and may lead to permanent exclusion from the Hacktoberfest event.

### Adding a new Hacktoberfest participant

- [ ] I did not edit the README.md file by hand
- [x] I have checked to ensure the new participant is not a duplicate and has not already been added
- [x] I am not adding a company from the [blocklist](../blob/master/.gitignore)
- [x] I have added the participant according to the [Contributing.md](../blob/master/CONTRIBUTING.md) guidelines
---
Hey, 
Happy Hacktoberfest! 🎉 
The list looked very empty 😄 so I added a bunch of them together.
Hope everything looks OK.

Thank you.

### CODE REVIEW

File: participants/dagshub.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name: DagsHub
+Website: https://dagshub.com/
+Swag: 
+  - swag 
+  - shirt
+  - stickers
+  - other
+Description: "Register to [DagsHub](https://dagshub.com/user/sign_up) and Hacktoberfest and contribute to any of the DagsHub hosted projectst with the topic 'hacktoberfest'."
```
Comment (jbampton) [inline participants/dagshub.yml:8]:
```suggestion
Description: "Register to [DagsHub](https://dagshub.com/user/sign_up) and Hacktoberfest and contribute to any of the DagsHub hosted projects with the topic 'hacktoberfest'."
```

File: participants/flyte.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name: Flyte
+Website: https://flyte.org/
+Swag: 
+  - shirt
+  - swag
+  - stickers
+  - other
+Description: "Make atleast 2 merged/accepted PRs to any of the issues labeled 'hacktoberfest'. Check [this issue](https://github.com/flyteorg/flyte/issues/4064) for all the eligible issues."
```
Comment (jbampton) [inline participants/flyte.yml:8]:
```suggestion
Description: "Make at least 2 merged/accepted PRs to any of the issues labeled 'hacktoberfest'. Check [this issue](https://github.com/flyteorg/flyte/issues/4064) for all the eligible issues."
```

**Comment by vimode** [comment]
@jbampton Thank you for the quick review. Fixed the typos.

**Comment by vimode** [comment]
Fixed the broken build status badge in the README.

File: participants/docsgpt.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+Name: DocsGPT
+Website: https://docsgpt.arc53.com/
+Swag:
+  - shirt
+Description: "Contribute to code and make a valid PR or improve the documentation by fixing the exisiting documentation or by creating a guide or improve the UI/UX or design a new feature."
```
Comment (jbampton) [inline participants/docsgpt.yml:5]:
```suggestion
Description: "Contribute to code and make a valid PR or improve the documentation by fixing the existing documentation or by creating a guide or improve the UI/UX or design a new feature."
```

**Comment by dishantzaveri** [comment]
Pls assign me @vimode

**Pull request merged**